packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -77,24 +77,25 @@ export class AtomicInsightSearchBox {
   public render() {
     return (
       <SearchBoxWrapper disabled={this.disableSearch}>
+        <atomic-icon
+          part="submit-icon"
+          icon={SearchIcon}
+          class="w-4 h-4"
+        ></atomic-icon>
```

Comment:
I guess no self-closing element in Web Component / Stencil?

Reply:
I don't think there are special limitations regarding self-closing elements. Will change that for a self-closing element. Thanks